mne/channels/montage.py

```diff
@@ -298,7 +298,7 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         if ext == '.hpts':
             fids = ('2', '1', '3')  # Alternate cardinal point names
         else:
-            fids = ('nz', 'lpa', 'rpa')
+            fids = ('nasion', 'lpa', 'rpa')
```

Comment:
I think that this will be fine. the current state is actually a deviation from the [docstring](https://github.com/mne-tools/mne-python/blob/master/mne/channels/montage.py#L158) on the name of the fiducials in the montage file for the transformation to work